lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -103,6 +130,7 @@ def step_navs_combined_list
         step_nav_list = []
         step_nav_list += step_navs if step_navs.any?
         step_nav_list += related_to_step_navs if related_to_step_navs.any?
+        step_nav_list += secondary_to_step_navs if secondary_to_step_navs.any?
```

Comment:
Adding `secondary_to_step_navs` here also means that `show_also_part_of_step_nav?` could start returning true as `also_part_of_step_nav` will also contain the secondary step nav.

I think we should seriously consider refactoring this module to make the step by step rules easier to understand.